### ISSUE TITLE
Question about dual boot and shutdown/hibernate (in windows)

### ISSUE THREAD

**Issue by thekswenson** [jakeday/linux-surface#230]
Thank you Jakeday for compiling and making this kernel usable on my Surface Pro 4.
No, this is not a question about the kenel, but instead about dual boot and hibernation when using windows (I need to use windows when taking notes or annotating a PDF with the pen since I haven't found stable enough software yet on the linux side, and I can't hiberate on the linux side (https://github.com/jakeday/linux-surface/issues/56)).

After a windows update things work normally on the windows side, meaning that the SP4 will shutdown and hibernate normally. After reinstating GRUB these things no longer work on the windows side.

Would anybody know if this is fixable?

**Comment by novo-redeunt** [comment]
Did you have a look at MasterPDF Editor? Works nice on Linux side (Kubuntu 18.04, Surface Book 2) - note taking and annotating with the pen are working nice. However, it is not open source.

**Comment by thekswenson** [comment]
I've had mixed luck in linux with:

 - Xournal++ (https://github.com/xournalpp/xournalpp) for annotating PDFs
 - Write (http://www.styluslabs.com/) for notetaking

For Xournal++, sometimes the pressure sensitivity works and sometimes it doesn't; sometimes it burns CPU and sometimes it doesn't.
Write is really a beautiful and innovative notetaking app (and quite stable). I'm very disappointed that pressure sensitivity doesn't work, so it's a bit like writing with a marker on a white board.  It has a version compiled for Windows that I use all the time (with pressure sensitivity).

Basically, the bugs in these two apps, along with the unusable state of hibernation for my SP4 is what denies me the gratification of erasing Windows. It's depressing using Windows on a daily basis.

The 70$ fee for a MasterPDF license smarts, but I'd consider it if I can get hibernate to work more than once in a while.

**Comment by thekswenson** [comment]
By the way, I found this link
  https://answers.microsoft.com/en-us/windows/forum/windows_10-power-winpc/windows-10-hibernate-back-from-hibernate/e70cfde9-4982-4c3d-be94-3459e6ece88b
which may have the answer.  Looks like I may need to have two bootloader, one for windows and the main one, and then another on the linux partition.

I'll report back when I get the time to fiddle with this.

**Comment by thekswenson** [comment]
I've had some time to look into this.
I'm able to reinstate the normal operation of Windows hibernation/shutdown by entering the UEFI configuration (choose "System Setup" in GRUB) and disabling the "ubuntu" item.

I can then use Ubuntu again by holding the power and volume buttons on boot (this adds a new ubuntu item in the UEFI configuration).

Needless to say, this is not ideal.

So what I'm trying to do is reinstate the "Windows Boot Manager" so that I can use it to choose either Windows or GRUB.  Does anyone know if this is possible?

I've tried to use EasyBCD but apparently it will not work since there is no "Legacy Boot" mode on the SP4.
https://neosmart.net/wiki/easybcd/uefi/

Any help would be appreciated.  Thanks!

**Comment by jakeday** [comment]
Please post these kinds of things on the surfacelinux subreddit.
